src/app/core/services/session/session.service.ts

```diff
@@ -121,4 +121,43 @@ export class SessionService {
       this.loaderService.stopLoader();
     }
   }
+
+  async joinSession(id){
+    await this.loaderService.startLoader();
+    const config = {
+      url: urlConstants.API_URLS.JOIN_SESSION+id,
+      payload: {}
+    };
+    try {
+      let data = await this.httpService.get(config);
+      this.loaderService.stopLoader();
+      if(data.responseCode == "OK"){
+        let browser = this.inAppBrowser.create(data.result.link,"_system");
```

Comment:
can we make this browser part a common function? As we are using the same in start session and join session